src/main/service/subscriptionService.ts

```diff
@@ -66,4 +68,23 @@ export class SubscriptionService {
     }
   }
 
+  public setPendingSubscriptions(searchResult, user): boolean {
+    pendingSubscriptionsFromCache.setPendingSubscriptions(searchResult, user);
+    return true;
```

Comment:
This function will always return "true". Is it correct?

Reply:
refactored